.changeset/tidy-goats-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
+---
+
+Reverted the placement of the Contextual Save Bar back to the original position over the global search. Deprecated the `sidebar` prop to the Frame component.
```

Comment:
Should we 🔥  this and just keep the deprecation notice here?

Reply:
Good point. Since this is only related to the feature flag, I think so 👍